src/tpm2tss-genkey.c

```diff
@@ -57,11 +57,12 @@ char *help =
     "    -o, --ownerpw   password for the owner hierarchy (default: none)\n"
     "    -p, --password  password for the created key (default: none)\n"
     "    -P, --parent    specific handle for the parent key (default: none)\n"
+    "    -W, --parentpw  password for the parent key (default: none)\n"
     "    -s, --keysize   key size in bits for rsa (default: 2048)\n"
```

Comment:
Please order alphabetically

Reply:
Ah, I didn't realize these were in alphabetical order. Will fix!